app/javascript/components/concept-exercise-graph/ExerciseGraph.css

```diff
@@ -0,0 +1,30 @@
+:root {
```

Comment:
So regarding the CSS:
- I'd like to see all the CSS for the graph in one file.
- It should follow the rules in the [Contributing guide](https://github.com/exercism/v3-website/blob/master/docs/CONTRIBUTING.md#css) - specifically regarding the `c-` prefix and nesting.
- We probably need to wait for Tai's final design before doing this "properly", but when we do, we'll be using Tailwind (a mixture of inline and `@apply`) to write the CSS. I can do this, or you can learn Tailwind as well 🙂 I'm **super**-picky about CSS as a pre-warning though 😉 

Chat to me more on Slack about this if you have questions 🙂

Reply:
> * I'd like to see all the CSS for the graph in one file.

Can do

> * It should follow the rules in the [Contributing guide](https://github.com/exercism/v3-website/blob/master/docs/CONTRIBUTING.md#css) - specifically regarding the `c-` prefix and nesting.

Sure

> * We probably need to wait for Tai's final design before doing this "properly", but when we do, we'll be using Tailwind (a mixture of inline and `@apply`) to write the CSS. I can do this, or you can learn Tailwind as well 🙂 I'm **super**-picky about CSS as a pre-warning though 😉

That's reasonable given this component